internal/api/helper.go

```diff
@@ -0,0 +1,67 @@
+package api
+
+import (
+	"net/http"
+	"strconv"
+
+	logger "github.com/sirupsen/logrus"
+)
+
+func getPaginationParams(req *http.Request) (page int16, limit int16) {
+
+	pageStr := req.URL.Query().Get("page")
+	limitStr := req.URL.Query().Get("page_size")
+
+	// if pageStr == "" {
+	// 	page = 1
+	// } else {
+	// 	pageInt64, err := strconv.ParseInt(pageStr, 10, 32)
+	// 	if err != nil {
+	// 		logger.Errorf("err: %v",err)
+	// 		page = 1
+	// 	}
+
+	// 	if pageInt64 < 1 {
+	// 		pageInt64 = 1
+	// 	}
+	// 	page = int16(pageInt64)
+	// }
+
+	page = 1
+	if pageStr != "" {
+		pageInt64, err := strconv.ParseInt(pageStr, 10, 32)
+		if err != nil {
+			logger.Errorf("err: %v", err)
+		}else if pageInt64 > 0 {
+			page = int16(pageInt64)
+		}
+	}
+
+	limit = 10
+
+	if limitStr != "" {
+		limitInt64, err := strconv.ParseInt(limitStr, 10, 32)
+		if err != nil {
+			logger.Errorf("err: %v", err)
+		}else if limitInt64 > 1000 {
```

Comment:
create and use MaxLimit const.

Reply:
created DefaultPageSize in constants
also table names are created in system.go file